tomviz/Pipeline.h

```diff
@@ -126,11 +135,15 @@ public slots:
   DataSource* findTransformedDataSource(DataSource* dataSource);
   Operator* findTransformedDataSourceOperator(DataSource* dataSource);
   void addDataSource(DataSource* dataSource);
+  bool beingEdited(DataSource* dataSource) const;
+  bool isModified(DataSource* dataSource, Operator** firstModified) const;
 
   DataSource* m_data;
   bool m_paused = false;
+  bool m_deletedOperators = false;
```

Comment:
I think this should the ```m_operatorsDeleted``` otherwise is sounds like is show stored the deleted operators.